commit 8c43d0347e3cc3445e636ede5d5c3f00e52b63b7 (refs/changes/73/1373/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-30T16:13:34-08:00 (2 years, 8 months ago)
    
    MANTA-3133 'manta-deploy-dev' should specify the server on which to setup manta-marlin

diff --git a/bin/manta-deploy-dev b/bin/manta-deploy-dev
index 4990894..adfb379 100755
--- a/bin/manta-deploy-dev
+++ b/bin/manta-deploy-dev
@@ -109,5 +109,5 @@ confirm "Are you ready to proceed? (y/N) " || fail "aborted by user"
 
 set -o xtrace
 manta-adm update -y $mdl_conffile
-manta-marlin
+manta-marlin -s $(manta-adm cn -H -o server_uuid storage)
 set +o xtrace
